### PR TITLE
Always use POSIX paths for ES imports

### LINKED ISSUE #15: The separator in import clause should be '/'
**Bug** 

When I use `@dojo/cli-create-widget` to generate a widget template, 
the generated widget import clause is error

```ts
import * as css from './styles\container.m.css';
```

the test class also has the same error

```ts
import Container from './..\..\Container';
```

My pc is Windows 11

### PR DESCRIPTION
**Pull request by bitpshr**
**Type:** bug

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
* [x] Unit or Functional tests are included in the PR

**Description:**

ES imports were botched on windows machines due to differences in the output of `path.relative` between win32 and POSIX operating systems. This PR modifies the usage of `path.relative` to use `path.posix.relative` since ES imports should always use POSIX path delimiters.

Resolves #15

### CODE REVIEW

**Comment by agubler** [comment]
@bitpshr Can we test this or it already covered? Looks like Appveyor is not setup correct for this repo so would never have run them on windows before.

**Comment by agubler** [comment]
@bitpshr Awesome setting up appveyor correctly and the previous commit on master [fails](https://ci.appveyor.com/project/Dojo/cli-create-widget).

Can you push a commit so that we get this PR to re-run on appveyor?

**Comment by bitpshr** [comment]
Thanks @agubler, ready for another review.

**Comment by agubler** [comment]
@bitpshr Cool, it doesn't look like that failing test was related to this change - which means we haven't covered the creation of these paths in a test (by covered I mean actually tested the values)?

Slightly off topic, we should probably add fixture tests for this repo to ensure that the output is actually correct! But we can do that as a follow up PR.

File: src/run.ts [hunk 1]
```diff
@@ -74,9 +74,9 @@ export default async function(helper: Helper, args: CreateWidgetArgs) {
 		name,
```
Comment (agubler) [inline src/run.ts]: @bitpshr We should be able to use `MockModules.ts`, to assert the arguments of `renderFiles` that will cover these changes.

I don't think we have mock modules in this repo, but you should be able to copy it from [@dojo/webpack-contrib](https://github.com/dojo/webpack-contrib/blob/master/tests/support/MockModule.ts) and then use it similar to the [build-time-render tests](https://github.com/dojo/webpack-contrib/blob/master/tests/unit/build-time-render/BuildTimeRender.ts#L27)

**Comment by bitpshr** [comment]
@agubler Thanks for the reviews so far, ready for another. We were already using `mockery` directly in this test and had `renderFiles` stubs set up already, so I just hooked into those for now.

**Comment by agubler** [comment]
Thanks @bitpshr

**Pull request merged**